src/markdown-pages/add-ons/longhorn.md

```diff
@@ -39,3 +39,10 @@ spec:
 ```
 
 To enable the UI without re-running the installer, use the command `kubectl -n longhorn-system scale deployment longhorn-ui --replicas=1`.
+
+## Migration from Rook
+
+If Rook was previously installed but no longer specified in the kURL spec, Longhorn 1.1.2+ will migrate data from Rook PVCs to Longhorn.
```

Comment:
```suggestion
If Rook was previously installed but is no longer specified in the kURL spec and Longhorn 1.1.2+ is specified instead, Longhorn will migrate data from Rook PVCs to Longhorn.
```
Mention that Longhorn is specified in its place, not just that Rook isn't specified.